README.md

```diff
@@ -1,12 +1,10 @@
 # Work_Zone
```

Comment:
@abinashkonersman any progress on this one?

Reply:
@payneBrandon i added that description after Google Cloud Function description and diagram.Please check and let me know if there is any comment.

Reply:
I added a bit to the combine description - take a look and let me know what you think. Can we similarly add some text talking about the logic to parse out iCone and COtrip data and what fields are currently supported? Ideally we want anyone to be able to pick up this repo, plug in an iCone data feed (or COtrip) and know exactly what the code is doing.

Reply:
@payneBrandon  we have [Word docs](https://github.com/CDOT-CV/Work_Zone/blob/CV-96/translator/docs/WZDx%20Translator%20Field%20Mappings.docx) that describes all the fields of iCone and Cotrip data feed.

Reply:
@payneBrandon  should be fixed i added a little detailed description.

Reply:
I like the link to our documentation that you added in. Next sprint we'll want to improve that documentation and update it a bit (we still reference spec 3 instead of 3.1 in a few places for instance).